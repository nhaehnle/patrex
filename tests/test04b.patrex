
match $<|before| $>|this| $( dolist( $!(,)*|obj|, $( $!(,)*|param| )+(,)[params] ); )|call|
	letrange gap before call.start
	erase gap
	erase call
	forall params
		insert call.start "{gap}do({obj}, {param});"
