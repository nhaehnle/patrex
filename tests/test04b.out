
	{
		do(obj, a);
		do(obj, b);
	}
